FirebasePerformance/Sources/AppActivity/FPRSessionManager+Private.h

```diff
@@ -25,10 +25,12 @@ NS_ASSUME_NONNULL_BEGIN
 @property(atomic, nullable, readwrite) FPRSessionDetails *sessionDetails;
 
 /**
- * Checks if the currently active session is beyond maximum allowed time. If so renew the session,
- * else no-op.
+ * Checks if the currently active session is beyond maximum allowed time for gauge-collection. If so
+ * stop gauges, else no-op.
  */
-- (void)renewSessionIdIfRunningTooLong;
+- (void)stopGaugesIfRunningTooLong:(NSDate *)now;
+
+- (BOOL)isGaugeCollectionEnabledForSessionId:(NSString *)sessionId;
```

Comment:
Add comments for this API as to what expect of this API.

Reply:
Done